common/service/mctp/mctp.c

```diff
@@ -51,14 +51,30 @@ static uint8_t set_thread_name(mctp *mctp_inst)
 	    mctp_inst->medium_type >= MCTP_MEDIUM_TYPE_MAX)
 		return MCTP_ERROR;
 
-	if (mctp_inst->medium_type == MCTP_MEDIUM_TYPE_SMBUS) {
+	switch (mctp_inst->medium_type) {
+	case MCTP_MEDIUM_TYPE_SMBUS:
+		printf("[%s] medium_type = SMBUS\n", __func__);
```

Comment:
Change to use Zephyr logging service instead of printf.

This would look something like:
```
LOG_INF(medium_type = SMBUS);
```

There are examples in other parts of the code on how to set that up.

Reply:
Complete modification